src/components/Tree/Tree.tsx

```diff
@@ -46,13 +47,18 @@ export const Tree = ({
   height,
   onCheck,
   onSelect,
+  selectable,
   showIcon,
   showLine,
   treeData,
   virtual,
 }: TreeProps): ReactElement => {
   return (
     <AntTree
+      // NOTE: This props force a controlled state to the component
+      // that is enabled also if its value is undefined, probably
+      // because the controlled mode is based exclusively on the props key definitions.
+      {...checkedKeys ? { checkedKeys } : {}}
```

Comment:
```suggestion
      // NOTE: The presence of this property forces a controlled state to the component
      // even if its value is undefined, probably
      // because the controlled mode is based exclusively on the props key definitions.
      {...checkedKeys ? { checkedKeys } : {}}
```